github/resource_github_repository_webhook.go

```diff
@@ -173,6 +186,12 @@ func resourceGithubRepositoryWebhookRead(d *schema.ResourceData, meta interface{
 		}
 	}
 
+	if hook.Config["insecure_ssl"] == "1" {
+		hook.Config["insecure_ssl"] = true
+	} else {
+		hook.Config["insecure_ssl"] = false
+	}
```

Comment:
This code should be pulled out to a function instead of repeating x3

Reply:
Done